README.md

```diff
@@ -14,3 +14,7 @@ Cluster DNS Operator manages CoreDNS as a Kubernetes DaemonSet exposed as a Serv
 ## How to help
 
 See [HACKING.md](HACKING.md) for development topics.
+
+## Reporting issues
+
+Bugs are tracked in [Bugzilla](https://bugzilla.redhat.com/enter_bug.cgi?product=OpenShift%20Container%20Platform).
```

Comment:
Preselect component and version? https://bugzilla.redhat.com/enter_bug.cgi?product=OpenShift%20Container%20Platform&version=4.0.0&component=Networking

Reply:
Nice, didn't know you could do that